ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -139,6 +139,33 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-efa-pg-f28
+  cron: 42 10 18 * *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-efa-pg
+- as: aws-ipi-localzone-byo-subnet-ovn-day2-f28
+  cron: 15 19 24 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn-day2
+- as: aws-ipi-localzone-byo-subnet-sdn-f28
```

Comment:
We can remove this one.
There is other job covered sdn. And aws-ipi-localzone-byo-subnet-ovn-day2-f28 covers other configuration except sdn.